<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="db" uuid="0676bfd9-abc3-4434-8a04-ed154c13db5c">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\Aurimas\Desktop\pythonas\5 uzd\invoice_application\db.sqlite3</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="d76f7d39-da00-48aa-be6f-8b09df5d29fa">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/invoice_application/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\Aurimas\Desktop\pythonas\5 uzd\invoice_application\db.sqlite3</jdbc-url>
    </data-source>
  </component>
</project>